db/migrate/20221217120959_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
```

Comment:
- [ ] Good job so far by creating tables 👍. I think you still have some issues to fix like the text should be `text` and not `string` 🙃, please kindly fix that 😉 
Both a string and text field will hold information that you can freely write in but the string field has a limit of 255 characters, whereas a text field has a character limit of 30,000 characters. So I think it would be better if for the text you use `text`